app/src/main/java/org/piwigo/accounts/UserManager.java

```diff
@@ -78,6 +80,8 @@
     private Cookie mSessionCookie;
     private String mSessionToken;
 
+    private CountingIdlingResource userIdleResource;
```

Comment:
Can you add a comment here when we should call more- or lessBusy and give a quick intro to the idling resource thing? Maybe 2-3 sentences to have something to start with and get the basic idea.